client/src/ui/molecules/main-menu/index.scss

```diff
@@ -23,7 +23,6 @@ ul.main-menu {
     display: flex;
     justify-content: space-around;
     margin: 0 auto 0 0;
-    max-width: 310px;
```

Comment:
@fiji-flo Can you explain this change?

Reply:
I don't know with this existed. I handles adding more entries in the main menu very ungraceful.